### ISSUE TITLE
File path in *.ds file

### ISSUE THREAD

**Issue by JinyuanSun** [rdk/p2rank#58]
Hi, 
It seems the file name in `*.ds` file does not support an absolute path like `/home/user/dataset/*.pdb`.
I have a `.ds` file:
```
/home/jsun/gpse_v2/test/structure_dataset/A0A1Q8LK65_AF.pdb
/home/jsun/gpse_v2/test/structure_dataset/W7ISS8_AF.pdb
/home/jsun/gpse_v2/test/structure_dataset/A5EDZ7_AF.pdb
/home/jsun/gpse_v2/test/structure_dataset/A0A3G7HEQ2_AF.pdb
/home/jsun/gpse_v2/test/structure_dataset/A0A6J4VWF3_AF.pdb
/home/jsun/gpse_v2/test/structure_dataset/A0A1R4K1N1_AF.pdb
/home/jsun/gpse_v2/test/structure_dataset/A0A7W8E001_AF.pdb
```
and in the error log:
```
[INFO] Console - ----------------------------------------------------------------------------------------------
[INFO] Console -  P2Rank 2.4
[INFO] Console - ----------------------------------------------------------------------------------------------
[INFO] Console - 
[INFO] Main - loading default config from [/home/jsun/gpse_v2/p2rank_2.4/config/default.groovy]
[INFO] Main - looking for dataset in dataset_base_dir [/home/jsun/gpse_v2/GPSE/filtered.ds]...
[INFO] Dataset - loading dataset [/home/jsun/gpse_v2/GPSE/filtered.ds]
[ERROR] Dataset - protein file doesn't exist: .//home/jsun/gpse_v2/test/structure_dataset/A0A7W8E001_AF.pdb
[ERROR] Dataset - protein file doesn't exist: .//home/jsun/gpse_v2/test/structure_dataset/W7ISS8_AF.pdb
[ERROR] Dataset - protein file doesn't exist: .//home/jsun/gpse_v2/test/structure_dataset/A0A1Q8LK65_AF.pdb
[ERROR] Dataset - protein file doesn't exist: .//home/jsun/gpse_v2/test/structure_dataset/A5EDZ7_AF.pdb
[ERROR] Dataset - protein file doesn't exist: .//home/jsun/gpse_v2/test/structure_dataset/A0A1M4DY30_AF.pdb
[ERROR] Dataset - protein file doesn't exist: .//home/jsun/gpse_v2/test/structure_dataset/A0A447WAH0_AF.pdb
[ERROR] Dataset - protein file doesn't exist: .//home/jsun/gpse_v2/test/structure_dataset/A0A1R4K1N1_AF.pdb
```
How can I modify the setting to avoid this? Thanks a lot! 🙏

**Comment by rdk** [comment]
Thanks for the bug report. This is a bug and unfortunately there is no workaround or setting that would fix it.

It fill be fixed in the next release. It's already fixed in develop branch so you can make your own build.

**Comment by rdk** [comment]
Fix will be included in the next release.

**Comment by rdk** [comment]
Fix released in [2.4.1](https://github.com/rdk/p2rank/releases/tag/2.4.1)
